nixos/modules/config/networking.nix

```diff
@@ -41,19 +41,6 @@ in
       '';
     };
 
-    networking.hostConf = lib.mkOption {
-      type = types.lines;
-      default = "multi on";
```

Comment:
Shouldn't we add an `/etc/host.conf` with this value by default?

Reply:
Good point. Will update the PR.

Reply:
Is `host.conf` honoured in the `nsswitch.conf` world?

Reply:
AFAICT yes as long as you use glibc's resolver directly. It might by another story when using dnsmasq or systemd-resolvd .

Reply:
This should happen inside the `file` part of `nsswitch.conf`, so before and regardless of which resolver is used to do DNS lookups.

For context, this was introduced during https://github.com/nixos/nixpkgs/issues/19148, and seems to be the default on ArchLinux.

Reply:
@flokli I've studied the man pages but got still no clue about how this should be expressed in nsswitch.conf. Could you provide an example?